library/hawkular_alerts_group.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
+
+
+DOCUMENTATION = '''
+---
+module: hawkular_alerts_group
+short_description: Creating Group Triggers in Hawkular Alerting
+requirements: [ hawkular/hawkular-client-python ]
+author: Daniel Korn (@dkorn)
+options:
+  hawkular_api_hostname:
+    description:
+      - the hawkular API hostname
+    default: HAWKULAR_HOSTNAME env var if set, otherwise it is required to pass it
+    required: True
+  hawkular_api_port:
+    description:
+      - the hawkular API port
+    default: HAWKULAR_PORT env var if set, otherwise it is required to pass it
+    required: True
+  hawkular_api_auth_token:
+    description:
+      - the hawkular API auth token
+    default: HAWKULAR_TOKEN env var if set, otherwise it is required to pass it
+    required: True
+  tenant:
+    description:
+      - the hawkular tenant
+    default: null
+    required: True
+  name:
+    description:
+      - the group trigger name
+    default: null
+    required: False
+  group_id:
+    description:
+      - the group trigger id
```

Comment:
Please document (here or elsewhere) that group_id is the "primary field" on which you match existing trigger.
(is there a convention in ansible documentation how/where to word this?)

It's good to always clarify this, but especially here both group_id and name could be reasonable behaviors.